File: csv/ocaml/create/csv_create.ml
(* ------------------------------------------------------------------- *)
(*
	csv_create.ml

					Aug/23/2011

*)
(* ------------------------------------------------------------------- *)
open	Printf
open	Text_manipulate
(* ------------------------------------------------------------------- *)
type unit = {name:string; population:int; date_mod:string}

(* ------------------------------------------------------------------- *)
let data_prepare_proc = fun () ->
	let dict_aa = Hashtbl.create 100
	in let _ = dict_append_proc dict_aa "t1271" "千葉" 31756 "1995-4-20"
	in let _ = dict_append_proc dict_aa "t1272" "勝浦" 83912 "1995-7-12"
	in let _ = dict_append_proc dict_aa "t1273" "市原" 12656 "1995-5-21"
	in let _ = dict_append_proc dict_aa "t1274" "流山" 83139 "1995-3-24"
	in let _ = dict_append_proc dict_aa "t1275" "八千代" 64325 "1995-8-18"
	in let _ = dict_append_proc dict_aa "t1276" "我孫子" 72698 "1995-9-8"
	in let _ = dict_append_proc dict_aa "t1277" "鴨川" 82375 "1995-2-17"
	in let _ = dict_append_proc dict_aa "t1278" "銚子" 25476 "1995-1-9"
	in let _ = dict_append_proc dict_aa "t1279" "市川" 47921 "1995-10-5"
	in
		dict_aa
(* ------------------------------------------------------------------- *)
let _ = print_endline "*** 開始 ***"

let dict_aa = data_prepare_proc ()

let _ = dict_display_proc dict_aa

let _ = let text_file = Sys.argv.(1)
	in
	printf "%s\n" text_file;
	csv_write_proc text_file dict_aa

let _ = print_endline "*** 終了 ***"

(* ------------------------------------------------------------------- *)

File: csv/go/create/csv_create.go
// ----------------------------------------------------------------
//
//	csv_create.go
//
//					May/20/2018
//
// ----------------------------------------------------------------
package main

import (
	"fmt"
	"os"
)

// ----------------------------------------------------------------
func data_prepare_proc () map[string](map[string]interface{}) {
	dict_aa := make (map[string](map[string]interface{}))

	dict_aa["t1271"] = unit_gen_proc ("千葉",81236,"1921-7-19")
	dict_aa["t1272"] = unit_gen_proc ("勝浦",49561,"1921-9-20")
	dict_aa["t1273"] = unit_gen_proc ("市原",62397,"1921-3-14")
	dict_aa["t1274"] = unit_gen_proc ("流山",76128,"1921-2-31")
	dict_aa["t1275"] = unit_gen_proc ("八千代",61459,"1921-6-24")
	dict_aa["t1276"] = unit_gen_proc ("我孫子",86123,"1921-1-10")
	dict_aa["t1277"] = unit_gen_proc ("鴨川",42937,"1921-4-19")
	dict_aa["t1278"] = unit_gen_proc ("銚子",78231,"1921-9-28")
	dict_aa["t1279"] = unit_gen_proc ("市川",25978,"1921-5-14")

	return (dict_aa)
}

// ----------------------------------------------------------------
func main () {

	fmt.Printf ("*** 開始 ***\n")

	file_csv := os.Args[1]

	dict_aa := data_prepare_proc ()

	csv_write_proc (file_csv,dict_aa)

	fmt.Printf ("*** 終了 ***\n")
}

// ----------------------------------------------------------------
